src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/ColumnDefinitionCollectionConverter.cs

```diff
@@ -0,0 +1,114 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Windows.Controls;
+using System.Windows.Markup;
+using System.Globalization;
+using MS.Internal;
+
+#pragma warning disable 1634, 1691  // suppressing PreSharp warnings
+
+namespace System.Windows.Controls
+{
+    internal class ColumnDefinitionCollectionConverter : TypeConverter
```

Comment:
This should be `sealed` unless there's an explicit reason. The .NET policy is to seal types unless there's demand for creating derives (or it makes sense), as you can always unseal them later but not the other way around.

Given this is internal right now, it should definitely be `sealed`.

Reply:
Done